test/e2e/s3-e2e-specs.js

```diff
@@ -0,0 +1,39 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import request from 'request-promise';
+import path from 'path';
+import { tempDir, fs, zip } from 'appium-support';
+import { uploadZip, deleteZip } from '../../lib/s3';
+import sinon from 'sinon';
+
+const {openDir} = tempDir;
+const {readFile, writeFile, mkdir} = fs;
+
+chai.should();
+chai.use(chaiAsPromised);
+
+if (process.env.AWS_ACCESS_KEY_ID && process.env.AWS_SECRET_ACCESS_KEY) {
```

Comment:
rather than not declaring the test unless these keys are set in the env, what about skipping the test if they're not?